README.md

```diff
@@ -9,3 +9,64 @@ GitHub pages
 
  * [100 parallel calls](perf-tests/core_api/stack_analysis/production_100_calls/stack_analysis_parallel_calls_100_threads.png)
  * [statistic](perf-tests/core_api/stack_analysis/production_100_calls/stack_analysis_parallel_calls_100_threads_concurrent_100_threads_min_max_avg_times.png)
+
+# FAQs
+
+## What is the overall archicture and flow graph of Fabric8-Anaytics system?
+
+Coming soon...
```

Comment:
Already in progress: suggested schema can be found here: https://drive.google.com/file/d/1Km7NsvL0IDv1jjwj4ElioXdDCSF_P3K5/view?usp=sharing

Reply:
I think we can add another issue to keep track of the architecture and flow graph. What do suggest ?

Reply:
@CermakM Added issue here: https://github.com/fabric8-analytics/fabric8-analytics.github.io/issues/2

Reply:
@tuxdna, Sure, I'll update the issue once I make a progress